code/script.js

```diff
@@ -218,108 +220,141 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// This function starts and restart the game.
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  
+  //What else should happen when we start the game? 
+  //   Clears the game board fron the letSee-page
+  winOrLose.style.display = 'none';
+  board.style.display = "flex";
+
+  generateBoard();
+  setSecret();
+  console.log(secret.name);
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// this sets the currentQuestion objects when selecting object in meny
 const selectQuestion = () => {
+    // This variable stores what option group (category) the question belongs to.
```

Comment:
Great that you've divided the pre-existing comments and placed them in connection to the part they are meant to refer to. Can't be too clear.